source/MaterialXTest/ShaderValid.cpp

```diff
@@ -883,13 +909,22 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
     // This will be imported in each test document below
     ioTimer.startTimer();
     mx::DocumentPtr dependLib = mx::createDocument();
-    loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, dependLib);
-#ifdef MATERIALX_BUILD_GEN_GLSL
-    if (options.runGLSLTests)
+    std::set<std::string> excludeFiles;
+    if (!options.runGLSLTests)
```

Comment:
Manually exclude certain files that we know of. So the implementations are restricted.